src/studio-home/card-item/CardItem.test.jsx

```diff
@@ -53,6 +54,21 @@ describe('<CardItem />', () => {
     const viewLiveLink = getByText(messages.viewLiveBtnText.defaultMessage);
     expect(viewLiveLink).toHaveAttribute('href', props.lmsLink);
   });
+
+  it('should render correct links for non-library course pagination', () => {
+    const props = studioHomeMock.archivedCourses[0];
+    const { getByText, getByTestId } = render(<RootWrapper {...props} isPaginated />);
+    const courseTitleLink = getByText(props.displayName);
+    expect(courseTitleLink).toHaveAttribute('href', `${getConfig().STUDIO_BASE_URL}${props.url}`);
+    const dropDownMenu = getByTestId('toggle-dropdown');
+    fireEvent.click(dropDownMenu);
+    const btnReRunCourse = getByText(messages.btnReRunText.defaultMessage);
+    expect(btnReRunCourse).toHaveAttribute('href', props.rerunLink);
+    const viewLiveLink = getByText(messages.viewLiveBtnText.defaultMessage);
+    expect(viewLiveLink).toHaveAttribute('href', props.lmsLink);
+    const editInStudioLink = getByText(messages.editStudioBtnText.defaultMessage);
+    expect(editInStudioLink).toHaveAttribute('href', props.cmsLink);
+  });
```

Comment:
could we add a tests for when the feature is off?

Reply:
This is covered in the previous test since doesn't have the prop  `isPaginated` [here](https://github.com/openedx/frontend-app-course-authoring/blob/17fe28a813cbebcbfaf971d9c9a8fd2e3c5898a3/src/studio-home/card-item/CardItem.test.jsx#L47)